sql/src/test/java/io/crate/integrationtests/GeoBackwardCompatibilityTest.java

```diff
@@ -46,7 +46,7 @@ public void testWithinQuery() throws IOException {
          *      p geo_point);
          * INSERT INTO legacy_geo_point (id, p) VALUES (1, 'POINT (10 10)');
          */
-        Path zippedIndexDir = getDataPath("/indices/bwc/legacy_geo_point.zip");
+        Path zippedIndexDir = getDataPath("/indices/bwc/geo_point_1_7_5.zip");
```

Comment:
I would name it after the ES version, because the code also checks for example for `> V_2_3_0`